pkg/rancher-desktop/assets/translations/en-us.yaml

```diff
@@ -4,6 +4,7 @@
 generic:
   add: Add
   back: Back
+  ok: Ok
```

Comment:
I think we've on OK as the correct spelling

<img width="511" alt="Screenshot 2023-11-28 at 8 52 39 AM" src="https://github.com/rancher-sandbox/rancher-desktop/assets/835961/997c6f63-80ca-4c4c-b5ed-d818ae867d56">

Reply:
Uhm, yes, of course! Thanks for the hint. I took a look at https://github.com/rancher-sandbox/rancher-desktop/pull/5281 and than I did it the wrong way :woman_facepalming: